### PR TITLE
MTL-1748 Update PoR to Align with NCN Behavior

### PR DESCRIPTION
**Pull request by rustydb**
## Summary and Scope

_Summarize what has changed. Explain why this PR is necessary. What is impacted? Is this a new feature, critical bug fix, etc?_

The NCN behavior is a little less explicit and allows some freedom in the selection of OS disks.

Additionally NVME is fundamentally supported with the MTL-1748 changes.

_Is this change backwards incompatible, backwards compatible, or a backwards compatible bugfix?_

Yes.

## Issues and Related PRs

_List and characterize relationship to Jira/Github issues and other pull requests. Be sure to list dependencies._

* Resolves [MTL-1747](https://jira-pro.its.hpecorp.net:8443/browse/MTL-1747)
* Change will also be needed in `release/1.2`

### CODE REVIEW

**Review by jacobsalmela — APPROVED** [review]

File: background/ncn_mounts_and_file_systems.md [hunk 1]
```diff
@@ -5,12 +5,6 @@ reference information for these disks, their partition tables, and their managem
 
 ### Topics:
```
Comment (mharding-hpe) [inline background/ncn_mounts_and_file_systems.md:6]:
```suggestion
## Topics
```

File: background/ncn_mounts_and_file_systems.md [hunk 2]
```diff
@@ -135,7 +41,6 @@ The table below represents all recognizable FS labels on any given management no
 | ❌ | ✅ | ❌ | `CONLIB` | `/run/lib-containerd` | Ephemeral | `25%` | ✅ | [MTL-892](https://jira-pro.its.hpecorp.net:8443/browse/MTL-892) [CASMINST-255](https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-255) | |
 | ✅ | ❌ | ❌ | `ETCDLVM` | `/run/lib-etcd` | Ephemeral | `32 GiB` | ✅ | [CASMPET-338](https://jira-pro.its.hpecorp.net:8443/browse/CASMPET-338) | |
 | ✅ | ❌ | ❌ | `K8SLET` | `/var/lib/kubelet` | Ephemeral | `25%` | ❌ | [MTL-892](https://jira-pro.its.hpecorp.net:8443/browse/MTL-892) [CASMINST-255](https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-255) | |
->>>>>>> 5b2ff4be6f... afix all JIRA links to new JIRA
 
 The above table's rows with overlayFS map their "Mount Paths" to the "Upper Directory" in the table below:
```
Comment (mharding-hpe) [inline background/ncn_mounts_and_file_systems.md:45]:
```suggestion
The above table's rows with `overlayFS` map their `Mount Paths` to the `Upper Directory` in the table below:
```

File: background/ncn_plan_of_record.md [hunk 3]
```diff
@@ -24,14 +24,25 @@ This document outlines the hardware necessary to meet CSM's Plan of Record (PoR)
 
 > **`NOTE:`** Several components below are necessary to provide redundancy in the event of hardware failure.
 
+Any of the disks may be used over the following busses:
```
Comment (mharding-hpe) [inline background/ncn_plan_of_record.md:27]: My understanding is that both "busses" and "buses" are technically correct, but that "buses" is considered the more standard spelling.

```suggestion
Any of the disks may be used over the following buses:
```

File: background/ncn_plan_of_record.md [hunk 4]
```diff
@@ -24,14 +24,25 @@ This document outlines the hardware necessary to meet CSM's Plan of Record (PoR)
...
+- SAS
+- SATA
+- NVME
+
+> **`NOTE:`** USB is implicitly excluded during disk selection and wiping, the NCN's depoyment code will wipe all disks if they are a RAID or in the above list. The manual wipes will exclude USB, but it is recommended to verify manual wipes are actually doing so.
```
Comment (mharding-hpe) [inline background/ncn_plan_of_record.md:32]:
```suggestion
> **NOTE:** USB is implicitly excluded during disk selection and wiping. The NCN's deployment code will wipe all disks if they are a RAID or in the above list.
> The manual wipes will exclude USB, but it is recommended to verify that the manual wipes are actually doing so.
```

File: background/ncn_plan_of_record.md [hunk 5]
```diff
@@ -24,14 +24,25 @@ This document outlines the hardware necessary to meet CSM's Plan of Record (PoR)
...
+
+The OS disks are chosen by selecting the first two disks from a list containing all the disks in order by smallest to largest.
```
Comment (mharding-hpe) [inline background/ncn_plan_of_record.md:34]:
```suggestion
The OS disks are chosen by selecting the smallest disks. Two disks are used for OS disks by default.
```

**Review by mharding-hpe — CHANGES_REQUESTED** [review]

**Review by mharding-hpe — APPROVED** [review]

**Comment by mharding-hpe** [comment]
Should this be backported to 1.2?

**Pull request merged**